### ISSUE TITLE
bug or unexpected behavior recently introduced

### ISSUE THREAD

**Issue by esheldon** [esheldon/ngmix#24]
Some time after commit 5efd6b394eece17738fdb99d870e100bc133b17c a bug was introduced, or some behavior changed that is affecting programs that use ngmix.

@beckermr there are some commits from you in this range as well.

**Comment by beckermr** [comment]
Can we run a test with the repo at each commit? You can use a binary search to do this faster. Just pick a commit in the middle and test it. If it works, then look later, if not, then look earlier.

**Comment by esheldon** [comment]
This is where using fixed seeds for everything would have helped.  In ngmixit the guesses are random but we are not using fixed seeds, so we need to do large scale tests that take a long time.  I'm finally learning this lesson.

**Comment by esheldon** [comment]
Not all commits need to be checked.  Here are the commits of interest, from oldest to newest

858916e16359e89291a87d4da70dbd15b97fe442 fix bugs in symmetrize
0649ad4ee7e319c44c768a4eb4b0e43868656df7  merge from becker
3914967062997eb6ffb659cdaea7886bc528ba3c remove whiten and symmetrize
14fde2d23d4d7e314d322c742a9713ef61993f2b use eric's dilation
0cfa94f9f028549d10a62386320733927a50d25e add option to guess from max fit

**Comment by esheldon** [comment]
The binary search is a good idea without prior information.  But there are also ones I suspect more than others.   I suspect the "add option to guess from max fit" and "merge from becker" (the latter mainly because it wasn't from me, and I may be using the new code incorrectly

**Comment by esheldon** [comment]
argh, 3914967 remove whiten and symmetrize has a bug

**Comment by esheldon** [comment]
I see it in 0649ad4 "merge from becker"

**Comment by esheldon** [comment]
next is to try 858916e "fix bugs in symmetrize"

**Comment by esheldon** [comment]
This was not a bug.

This was the result the bug that was fixed here: 3f2dbe454049b389028e6886e5ea5f6a5c718ec7

I had been running sims with that bug in place, and it turns out that bug resulted in a very high variance in the metacal "Rpsf" parameter.  This was because for negative metacal shears, the psf was not dilated, and this resulted in larger variance of Rpsf overall.  This variance as not accounted for in my errors.

I got "lucky" with one of my very first deep-field runs that a fluctuation landed in such a way that the corrections were essentially perfect.

Now that this bug is fixed there is very small variance in Rpsf and I'm seeing more typical bias of order 1.0e-3 instead of 1.0e-4 and this had caused the confusion.
